wgsl/index.bs

```diff
@@ -2977,11 +2990,15 @@ TODO: deduplicate these tables
 `vec4<type> textureLoad(texture_ro, i32 coords, i32 level_of_detail)`
 `vec4<type> textureLoad(texture_ro, vec2<i32> coords, i32 level_of_detail)`
 `vec4<type> textureLoad(texture_ro, vec3<i32> coords, i32 level_of_detail)`
-  %32 = OpImageFetch %v4float %texture_ro %coords Lod %level_of_detail
+  %32 = OpImageLoad %v4float %texture_ro %coords Lod %level_of_detail
```

Comment:
Shouldn't this be OpImageRead?

Reply:
Oh yes, that's a mistake. Would you want to make a PR fixing this?

Reply:
I made one.